### ISSUE TITLE
`upstream` CI report failing

### ISSUE THREAD

**Issue by jrbourbeau** [dask/dask#9818]
It looks like our `upstream` CI job is failing during the step where it creates/updates an issue on GitHub (see [this build](https://github.com/dask/dask/actions/runs/3885576745/jobs/6629627686) for an example)

```python
 Parsing logs ...
Traceback (most recent call last):
  File "/home/runner/work/_actions/xarray-contrib/issue-from-pytest-log/v1.2.4/parse_logs.py", line 217, in <module>
    preformatted = [preformat_report(report) for report in failed]
  File "/home/runner/work/_actions/xarray-contrib/issue-from-pytest-log/v1.2.4/parse_logs.py", line 217, in <listcomp>
    preformatted = [preformat_report(report) for report in failed]
  File "/usr/share/miniconda3/envs/test-environment/lib/python3.10/functools.py", line 889, in wrapper
    return dispatch(args[0].__class__)(*args, **kw)
  File "/home/runner/work/_actions/xarray-contrib/issue-from-pytest-log/v1.2.4/parse_logs.py", line 89, in _
    message = report.longrepr.chain[0][1].message
AttributeError: 'str' object has no attribute 'chain'
```

Note we're using the latest `xarray-contrib/issue-from-pytest-log@v1.2.4` release. cc @keewis for visibility

**Comment by keewis** [comment]
I just released version `v1.2.5` of the action, which should fix this.

**Comment by jrbourbeau** [comment]
Awesome, thanks for fixing so quickly @keewis. Updating to 1.2.5 in https://github.com/dask/dask/pull/9822
